ci-operator/jobs/redhat-performance/jetlag/redhat-performance-jetlag-main-presubmits.yaml

```diff
@@ -5,7 +5,7 @@ presubmits:
     branches:
     - ^main$
     - ^main-
-    cluster: build05
```

Comment:
👀

Reply:
yes, no idea why. `make jobs` did it and ordered-prow-config was failing without that

Reply:
Sometimes this goes away when we rebase against latest master. Let's see if it will conflict...

Reply:
There was some problem with `build06` yesterday so maybe they have manually changed it to `build05` and now the `make jobs` is putting back `build06`. No idea.